bin/lib/options.js

```diff
@@ -46,7 +54,7 @@ module.exports = [
       'WebID-TLS',
       'WebID-OpenID Connect'
     ],
-    prompt: true,
+    prompt: false,
```

Comment:
Because at this point in the PR, there's only one authentication method. So there's no point in prompting for the choice.